net/mptcp/mptcp_sched.c

```diff
@@ -104,6 +149,64 @@ static int mptcp_dont_reinject_skb(const struct tcp_sock *tp, const struct sk_bu
 		mptcp_pi_to_flag(tp->mptcp->path_index) & TCP_SKB_CB(skb)->path_mask;
 }
 
+static bool subflow_is_selected(struct tcp_sock *tp, struct sk_buff *skb,
+				enum sk_groups sk_group)
+{
+	switch (sk_group) {
+	case ACTIVE_NOT_USED:
+		if (!subflow_is_backup(tp) && !mptcp_dont_reinject_skb(tp, skb))
+			return true;
+		return false;
```

Comment:
You could simply do:
return !subflow_is_backup(tp) && !mptcp_dont_reinject_skb(tp, skb);

Instead of the if-statement. The same for the other if-clauses.